drivers/display/display_ili9xxx.c

```diff
@@ -343,6 +343,13 @@ static int ili9xxx_configure(const struct device *dev)
 		return r;
 	}
 
+	if (config->inversion) {
```

Comment:
question: should inversion be called after `regs_init_fn`? does it produce a glitch otherwise?

Reply:
Does not produce any glitch, the display is blanked for the whole time. For the command sequence, I don't think it makes any difference, the controller datasheet does not seem to mention any sequence for configuration commands either so I think either works.